scripts/lint/ourlint.js

```diff
@@ -82,6 +82,19 @@ const TESTS = {
       )}`;
     }
   },
+
+  /* Check icon colors */
+  checkColor: (data) => {
+    const hasWhiteColor = (color) => {
+      return color === 'FFFFFF';
+    };
+
+    const allColors = [...new Set(data.icons.flatMap((icon) => icon.hex))];
+
+    if (allColors.some((color) => hasWhiteColor(color))) {
+      return 'Some icon colors are white `FFFFFF`, please replace with black `000000`';
```

Comment:
> please replace with black `000000`

I would not ask specifically to replace with black since it might not be the best alternative to represent the brand

Reply:
@LitoMore - I don't think we explicitly need to tell users to replace with black. Perhaps we just suggest they find a more appropriate colour?

```suggestion
      return 'Some icon colors are white `FFFFFF`, are you sure that is the correct color?';
```